templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl

```diff
@@ -75,6 +75,30 @@
                 "order": 3
             },
             "internalConsoleOptions": "neverOpen",
+        {{#CEAEnabled}}
+        },
+        {
+            "name": "Launch Remote in Copilot (Edge)",
```

Comment:
Is the wording (in Copilot) confirmed with Junjie?

Reply:
Frank has asked Junjie, waiting for his reply